content/en/docs/hertz/tutorials/service-governance/service_discovery/eureka.md

```diff
@@ -9,7 +9,7 @@ description: "Service Registration and Discovery eureka Extensions provided by H
 ## Install
 
 ```go
-go get github.com/hertz-contrib/eureka
+go get github.com/hertz-contrib/registry
```

Comment:
`github.com/hertz-contrib/registry/eureka`

Reply:
Thanks for reminder.